### PR TITLE
Lhapdf6 1 2

### PR DESCRIPTION
**Pull request by mkirsano**

### CODE REVIEW

**Comment by mkirsano** [comment]
This is the attempt of the first integration of lhapdf6. This one replaces PR #751 substituting version 6.1.0 by 6.1.2 and using the updated PDF sets in the CMS area on cvmfs (subdirectory /6.1.2).
 In order to use it in CMSSW one has to substitute lhapdf by lhapdf6 in the file(s) BuildFile. For example, if one wants that pythia8 uses lhapdf6, the file to change is GeneratorInterface/Core/BuildFile.

**Comment by bendavid** [comment]
bump

File: lhapdf6.spec [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+%define cms_cxxflags -O2 -std=c++0x
+%endif
+
+%prep
+%setup -q -n %{n}/%{realversion}
+
+./configure --prefix=%{i} --with-boost=${BOOST_ROOT} --with-yaml-cpp=${YAML-CPP_ROOT} PYTHON=${PYTHON_ROOT}/bin/python CYTHON=${CYTHON_ROOT}/bin/cython PYTHONPATH=${CYTHON_ROOT}/lib/python@PYTHONV@/site-packages
+
+%build
+make all PYTHONPATH=${CYTHON_ROOT}/lib/python@PYTHONV@/site-packages
```
Comment (ktf) [inline lhapdf6.spec:29]: Any reason why you do not have %makeprocesses (to build with -j)?

File: lhapdf6_makeLinks.file [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (ktf) [inline lhapdf6_makeLinks.file:1]: Please in general do not use /bin/bash, but /bin/sh.

**Comment by ktf** [comment]
Comments are just FYI, I'll fix them on my side.

**Comment by ktf** [comment]
Slightly fixed in #819. Closing this.

**Pull request closed without merging**